src/panels/lovelace/common/create-card-element.js

```diff
@@ -0,0 +1,78 @@
+import computeDomain from '../../../common/entity/compute_domain.js';
+
+import '../cards/hui-camera-preview-card.js';
+import '../cards/hui-entities-card.js';
+import '../cards/hui-entity-filter-card.js';
+import '../cards/hui-glance-card';
+import '../cards/hui-history-graph-card.js';
+import '../cards/hui-markdown-card.js';
+import '../cards/hui-media-control-card.js';
+import '../cards/hui-entity-picture-card.js';
+import '../cards/hui-picture-glance-card';
+import '../cards/hui-plant-status-card.js';
+import '../cards/hui-weather-forecast-card';
+import '../cards/hui-error-card.js';
+
+const CARD_TYPES = [
+  'camera-preview',
+  'entities',
+  'entity-filter',
+  'entity-picture',
+  'glance',
+  'history-graph',
+  'markdown',
+  'media-control',
+  'picture-glance',
+  'plant-status',
+  'weather-forecast'
+];
+
+const DOMAIN_DEFAULT_CARD = {
+  camera: 'camera-preview',
+  history_graph: 'history-graph',
+  media_player: 'media-control',
+  plant: 'plant-status',
+  weather: 'weather-forecast'
+};
+
+const CUSTOM_TYPE_PREFIX = 'custom:';
+
+export default function createCardElement(config) {
+  if (typeof config === 'string' && Object.keys(DOMAIN_DEFAULT_CARD).includes(computeDomain(config))) {
```

Comment:
Adding a config shortcut means that we are going to have to deal with this shortcut forever. So in that case, we have to think if that is something that we want. It's only for 5 different cards, and from how I see things going, I can see that it will be the minority of cards that can leverage this. I suggest that we do not implement this.

Reply:
- I think we will get more domain specific cards, e.g. for climate or alarm_panel
- for state-cards (`entities:`) we also allow this short syntax

my idea was also to use `typeof config === 'string' && config === 'separator'` in entities card to add a `<hr>` (that will now be to elements[])

Reply:
For state cards, we need to think more customizable. What about custom seperators ? We should probably create a `createSeperatorElement`. Allow a picture seperator so I can have these fancy decorators:

![blog-divider8](https://user-images.githubusercontent.com/1444314/41876145-e1ae72c8-789a-11e8-9cb6-b98b29a95f03.png)